--- a/common.mk
+++ b/common.mk
@@ -133,11 +133,11 @@ LIBSN_CFLAGS := $(call cflags_for_lib, libstartup-notification-1.0)
 LIBSN_LIBS   := $(call ldflags_for_lib, libstartup-notification-1.0,startup-notification-1)
 
 # Pango
-PANGO_CFLAGS := $(call cflags_for_lib, cairo)
-PANGO_CFLAGS += $(call cflags_for_lib, pangocairo)
-I3_CPPFLAGS  += -DPANGO_SUPPORT=1
-PANGO_LIBS   := $(call ldflags_for_lib, cairo)
-PANGO_LIBS   += $(call ldflags_for_lib, pangocairo)
+# PANGO_CFLAGS := $(call cflags_for_lib, cairo)
+# PANGO_CFLAGS += $(call cflags_for_lib, pangocairo)
+# I3_CPPFLAGS  += -DPANGO_SUPPORT=1
+# PANGO_LIBS   := $(call ldflags_for_lib, cairo)
+# PANGO_LIBS   += $(call ldflags_for_lib, pangocairo)
 
 # libi3
 LIBS = -L$(TOPDIR) -li3 -lm
